src/services/expenses/expenses.services.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @param {number} id*
+ * @param {number} userId*
+ * @param {Date} spentAt*
+ * @param {string} title*
+ * @param {number} amount*
+ * @param {string} category*
+ * @param {string} note
+ *
+ *
+ */
+
+const { getNewExpenseData } = require('../../untils/expenses');
+
+// const expenseData = {
+//   spentAt: '2022-10-19T11:01:43.462Z',
+//   title: 'Buy a new laptop',
+//   amount: 999,
+//   category: 'Electronics',
+//   note: 'I need a new laptop',
+// };
+
+// let expenses = Array.from({ length: 6 }, (_, i) => ({
+//   ...expenseData,
+//   userId: i,
+//   id: i,
+// }));
+
+let expenses = [];
+
+let lastId = expenses.length;
+
+const getExpenses = (categories, userId, from, to) => {
+  let filtredExpenses = [...expenses];
+
+  if (categories && Array.isArray(categories)) {
+    filtredExpenses = filtredExpenses.filter(({ category }) => {
+      return categories.includes(category);
+    });
+  } else if (userId) {
+    filtredExpenses = filtredExpenses((expense) => expense.userId === userId);
```

Comment:
There is a syntax error here. The method `filtredExpenses` should be `filter` instead of `filtredExpenses`. This will cause a runtime error.